Dev/pilgrimage events fixes (#215)
ProZeratul/CrusaderKings3UnofficialPatch: events/activities/pilgrimage_activity/pilgrimage_events.txt

	faith = root.faith
	NOT = { is_courtier_of = root }
	location = root.location

> 	NOT = { is_ruler = yes } #Unop To fix [ Trying to put character in court despite being a ruler ] (because of this char being taken in piligrimage travel)

Reviewer: ```suggestion
	is_ruler = no #Unop To fix [ Trying to put character in court despite being a ruler ] (because of this char being taken in piligrimage travel)
```
Nit
